examples/commander_bot.json

```diff
@@ -84,8 +84,12 @@
     "cogbot.extensions.invite": {
       "database": "https://raw.githubusercontent.com/MinecraftCommands/commanderbot/master/config/invite.json"
     },
-    "cogbot.extensions.mcnbtpaths": {
-      "database": "https://raw.githubusercontent.com/MrYurihi/mc-nbt-paths/master/output.json"
+    "cogbot.extensions.mcnbtdoc": {
+      "version_database": "https://raw.githubusercontent.com/MrYurihi/mc-nbtdoc/{}-gen/build/generated.json",
+      "database": "https://raw.githubusercontent.com/MrYurihi/mc-nbtdoc/generated/build/generated.json",
+      "ac_limit": 600,
+      "search_limit": 20,
+      "field_limit": 10
```

Comment:
Can you explain what those last 3 numbers control?

Reply:
ac limit is the time until an active embed deactivates
search limit is the limit to the entries on a `-ns` query
field limit is the limit to the number of fields in an embed (and in the `-s` query)